torch/quantization/fx/quantize.py

```diff
@@ -438,6 +438,11 @@ def load_arg(a):
                 cur_placeholder_node_idx = placeholder_node_seen_cnt
                 placeholder_node_seen_cnt += 1
                 if cur_placeholder_node_idx in input_quantized_idxs:
+                    # If an input node is marked as quantized, it was observed
+                    # elsewhere. For now, make it as observed for the purposes
+                    # of this file.  If in the future we will need the observer
+                    # instance for anything, we will have to adjust the API
+                    observed_node_names_set.add(node.name)
```

Comment:
oh, should this be merged with previous pR

Reply:
I think we should always do this. What are the cases that needs observer that might come up in the future?

Reply:
> I think we should always do this. What are the cases that needs observer that might come up in the future?

np, removed the comment

> oh, should this be merged with previous pR

I don't feel too strongly, but technically this is a separate bug and fix.  The previous PR removes the observer on the input of the first node, if the input is quantized.  This PR removes the observer on the output of the first node, if the input is quantized and the node does not need observation.  Lmk if you prefer them to be combined.

Reply:
sg, I don't have a strong preference either.